integrations/extensions/starter-kits/search-with-milvus/search-with-watsonx-data-milvus.md

```diff
@@ -0,0 +1,109 @@
+# How to set up watsonx Assistant search integration with watsonx.data Milvus
+This is a documentation about how to set up a search integration with the Milvus service from watsonx.data for using search or conversational search feature in watsonx Assistant.
+
+## Before you begin
+1. Provision an watsonx.data instance
+    * watsonx.data on IBM Cloud: https://cloud.ibm.com/watsonxdata
+    * watsonx.data on Cloud Pak for Data (on-prem): follow [this doc](https://www.ibm.com/docs/en/cloud-paks/cp-data/5.0.x?topic=services-watsonxdata) to install and set up watsonx.data on Cloud Pak for Data 
+2. Add an Milvus service in the watsonx.data console
+    * watsonx.data on IBM Cloud: [Adding a Milvus service](https://cloud.ibm.com/docs/watsonxdata?topic=watsonxdata-adding-milvus-service)
+    * watsonx.data on Cloud Pak for Data (on-prem): [Adding a Milvus service](https://www.ibm.com/docs/en/watsonx/watsonxdata/2.0.x?topic=milvus-adding-service)
+
+## Table of contents
+* [Step 1: Collect Milvus connection info](#step-1-collect-milvus-connection-info)
+  * [Get the credentials](#get-the-credentials)
+  * [Get other connection details](#get-other-connection-details)
+* [Step 2: Ingest data into Milvus](#step-2-ingest-data-into-milvus)
+  * [Option 1: Ingest data through watsonx.ai](#option-1-ingest-data-through-watsonxai)
+  * [Option 2: Ingest data using custom code](#option-2-ingest-data-using-custom-code)
+* [Step 3: Connect to watsonx Assistant](#step-3-connect-to-watsonx-assistant)
```

Comment:
When you say watsonx Assistant, please either say watsonx Orchestrate/Assistant or say Assistant Builder (which is the same in both products).